### PR TITLE
Adding Google Cloud SQL variants for mysql, sqlserver and postgres

### PR DESCRIPTION
**Pull request by dyaffe**
**Description:**

Adding Google Cloud SQL variants for mysql, sqlserver and postgres

This is required for their new partner program.

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/connectors/1016)

### CODE REVIEW

File: materialize-sqlserver/VARIANTS [hunk 1]
```diff
@@ -0,0 +1,2 @@
+materialize-google-cloud-sql-mysql
```
Comment (willdonnelly) [inline materialize-sqlserver/VARIANTS:1]: This looks like a copy-paste error?

**Review by willdonnelly — CHANGES_REQUESTED** [review]
LGTM except I think there's a mistake in the `materialize-sqlserver` variants list which needs to be fixed.

File: materialize-sqlserver/VARIANTS (hunk 1 above)
Comment (dyaffe) [reply materialize-sqlserver/VARIANTS]: Fixed that

**Review by willdonnelly — APPROVED** [review]
LGTM

**Pull request merged**